inference-engine/thirdparty/clDNN/kernel_selector/core/common/jitter.cpp

```diff
@@ -738,6 +743,12 @@ JitConstants MakeActivationJitConstants(ActivationFunction activation_function,
                     (input / (one + exp(neg(input)))).str()));
             break;
         }
+        case ActivationFunction::MISH: {
+            jitConstants.AddConstant(MakeJitConstant(
+                    macro_def,
+                    (input * tanh(log(one + exp(input)))).str()));
```

Comment:
Please take a look at https://github.com/opencv/opencv/pull/17624. There is better approximation for Mish

Reply:
@Lyamin-Roman, Do you have some numbers which of the implementations are faster?